supplementary/supplementary_tests/include/test_supplementary.h

```diff
@@ -116,6 +120,8 @@ class AlicaTestMultiAgentFixture : public AlicaTestMultiAgentFixtureBase
         ros::NodeHandle nh;
         std::string path;
         nh.param<std::string>("rootPath", path, ".");
+        spinner = std::make_unique<ros::AsyncSpinner>(4);
+        spinner->start();
```

Comment:
Supplementary tests are using AlicaRosCommunication. Do start and stop at appropriate places like turtlesim demo.

Reply:
Missed it. Will fix this.